System/Yubikey.30m.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+#
+# <xbar.title>YubiKey TOTP to clipboard</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>jebabin</xbar.author>
+# <xbar.author.github>jebabin</xbar.author.github>
+# <xbar.desc>generates totp codes using YubiKey and copy to clipboard, supports touch and touchless code generation</xbar.desc>
+# <xbar.dependencies>bash,perl</xbar.dependencies>
+# <xbar.version>1.2</xbar.version>
```

Comment:
Can you take a screenshot of the plugin in action and include the `xbar.image` tag please?

Reply:
Done